packaging/linux/run_keybase

```diff
@@ -154,8 +154,9 @@ init() {
   logdir="${XDG_CACHE_HOME:-$HOME/.cache}/keybase"
   runtime_dir="${XDG_RUNTIME_DIR:-$HOME/.config}/keybase"
 
-  warn_if_unwritable "$logdir"
-  warn_if_unwritable "$runtime_dir"
+  warn_if_exists_and_unwritable "$logdir"
```

Comment:
If these things don't exist yet, should we be making sure that their parent directory is writable too, so that Keybase will be able to successfully create them when the time comes?

Reply:
I don't think that has ever happened (~/.config or ~/.cache being unwritable). There's some bug I can't figure out that's causing Keybase to make these directories in the user's home directory when run with root which is causing the problem, that's what I wanted to address here. But I can add it too if you think it'll be helpful

Reply:
Meh I guess if it's never happened, it's not worth checking.  Just thought maybe in some kind of fresh environment, if `sudo run_keybase` was the first thing they did, they might have created those parent directories with bad ownership.  Anyway, I'm fine without it.